Source/Fuse.Common/Tests/FuseTest/ObservableCollector.uno

```diff
@@ -55,7 +55,7 @@ namespace FuseTest
 			CleanSubscription();
 			if (_items == null)
 				return;
-			_subscription = _items.Subscribe(this);
+			_subscription = (ISubscription)_items.Subscribe(this);
```

Comment:
This test has to assume this, because it is testing writebacks to observables. It will throw an exception otherwise.